coherence/upnp/core/service.py

```diff
@@ -366,6 +366,14 @@ def gotPage(x):
                                                                'n/a',
                                                                instance, send_events,
                                                                data_type, values)
+                allowed_range = var_node.find('{%s}allowedValueRange' % ns)
+                if allowed_range:
+                    allowed_range_dict = {}
+                    for item in allowed_range:
+                        namespace_uri, tag = item.tag[1:].split("}", 1)
+                        if tag in ['maximum','minimum','step']:
```

Comment:
Thanks for the fix.

Please follow PEP 8 style and add a space behind the comma.

Otherwise looks got to me